core/lib/globals_shared.h

```diff
@@ -1870,10 +1870,16 @@ typedef union _dr_simd_t {
 } dr_simd_t;
 #    endif
 #    ifdef X64
+#        define MCXT_NUM_SIMD_SSE_AVX_SLOTS \
+            32 /**< Unused in ARM/AArch64   \
+                */
```

Comment:
Seems like this can easily fit on one line and be more readable and streamlined then.  At least put the trailing `*/` up on the prior line.

Ditto below: looks awkward with the split `*/`.  Maybe move these all to before-comments.

Reply:
Done.